src/prefect/engine/state.py

```diff
@@ -810,6 +810,23 @@ class TriggerFailed(Failed):
     color = "#ff5131"
 
 
+class ValidationFailed(Failed):
```

Comment:
I just realized somewhere along the way I started calling this `ValidationFailed` though the PIN specifies `ValidationError`. Semantically I think they are similar but between the two I am for `ValidationFailed` since it matches the patterns of `TriggerFailed`/`ClientFailed`

Reply:
👍 I'm cool with it